boards/arm/frdm_k64f/frdm_k64f.dts

```diff
@@ -241,7 +241,7 @@ zephyr_udc0: &usbotg {
 		     &RMII0_MDC_PTB1>;
 	pinctrl-names = "default";
 	ptp {
```

Comment:
A comment over here about the pin conflict will be helpful

Reply:
ptc16/17 is use as tmr event for compare and capture, which is not a must in 1588. so we can disable the pinmux in pinmux.c instead of disable ptp in frdmk64f board.